prototype1/prototype_v1_JC_example.yml

```diff
@@ -0,0 +1,29 @@
+author:
+  -
+    string-name:
+    surname: Colomb
+    given-names: Julien
+    email: julien.colomb@fu-berlin.de
+    PID:
+      type: orcid-id
+      id: https://orcid.org/0000-0002-3127-5520
+    role:
+      Conceptualization: created first prototype
+    affiliation:
+      ref1
+    part-of-group: NA   
+    attributes:
+      is.corresponding.author: yes
+      type: author
+    
+Affiliation:
+  - refid: ref1
```

Comment:
Do we really need these affiliation IDs? I mean, institutions can have IDs just as ORCID for authors, right? If so, we could fall back to such arbitrary IDs only in case where an institution does not have an ID, and then we could create such an ID ourselves, or?

Citing from the recommendations:

> Capturing the institutional ID is not mandatory at this time

Reply:
That `refid` is only used to link authors with affiliation and created internally. It is needed if we keep affiliation and other author information separate and that the link (author-affiliation) cannot be derived from where the affiliation information is entered.
In practice, authors often share affiliation and writing the affiliation in the author information means any tool need to check for duplicates to link each affiliation only once.

Also affiliations ID only link to the universities, not to the department, so 2 different affiliations can have the same institution ID.

Reply:
basically #19